### ISSUE TITLE
Cannot use salt-bootstrap on fresh CentOS 7.0.1406

### ISSUE THREAD

**Issue by endragor** [saltstack/salt-bootstrap#456]
```
# sudo find /etc/ -name '*-release' -print -exec cat {} \;
/etc/centos-release
CentOS Linux release 7.0.1406 (Core) 
/etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

/etc/redhat-release
CentOS Linux release 7.0.1406 (Core) 
/etc/system-release
CentOS Linux release 7.0.1406 (Core)
```

Error in the output of `curl -L https://bootstrap.saltstack.com/develop | sudo sh -s -- git develop`:

```
...
 *  INFO: Running install_centos_git_deps()
curl: (22) The requested URL returned error: 404 Not Found
error: skipping http://download.fedoraproject.org/pub/epel/7/x86_64/epel-release-7-1.noarch.rpm - transfer failed
Retrieving http://download.fedoraproject.org/pub/epel/7/x86_64/epel-release-7-1.noarch.rpm
 * ERROR: Failed to run install_centos_git_deps()!!!
```

As it turns, the packages for version 7 should be taken from **beta** folder: `http://download.fedoraproject.org/pub/epel/beta/7/x86_64/epel-release-7-1.noarch.rpm`

**Comment by cvedel** [comment]
It looks like EPEL has added one extra directory level in their repository for 7. The correct URL is:  `http://download.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-1.noarch.rpm`

**Comment by pedropaulovc** [comment]
Should be fixed in `develop`, see PR #458

**Comment by s0undt3ch** [comment]
Closing as fixed.
